tsuru/client/services.go

```diff
@@ -711,6 +705,21 @@ func (c *ServiceInstanceRemove) Run(ctx *cmd.Context, client *cmd.Client) error
 	return nil
 }
 
+func getErrorAndMessageInSimpleJSON(jsons []string) (string, error) {
```

Comment:
Is there a reason we're not simply using `cmd.StreamJSONResponse` like every other command that stream output? I think using it would allow us to remove all this messy manual handling of the response json as it already returns any `Error` entry found in the stream.